Dockerfile

```diff
@@ -1,35 +1,17 @@
-FROM debian:stretch-slim
+ARG SRC_IMAGE=dashpay/dashd
+ARG SRC_IMAGE_TAG=latest
```

Comment:
Better put a specific version here to prevent unexpected upgrades.

Reply:
Version is to be overridden via arg on the dockerhub side using build hooks. We add git tag and push it and it triggers dockerhub build with image tag same as git tag and using upstream docker image with the same tag too. Check [this line](https://github.com/blockchain-etl/docker-dashd/pull/1/files#diff-14bdd6aac779ac93a185193b26ac4fd3R6), latest is for default builds only.

Reply:
Ok got it, thanks!